share/patch/patch_db_zonemaster_backend_ver_8.2.0.pl

```diff
@@ -54,7 +54,7 @@ sub patch_db_mysql {
         $dbh->do(
             q[
                 UPDATE test_results
-                SET domain = TRIM( TRAILING '.' FROM domain )
+                SET domain = LEFT( domain, LENGTH(domain) - 1 )
                 WHERE domain != '.' AND domain LIKE '%.'
```

Comment:
This code will create a valid domain from an invalid. I think the code should be
```
SET domain = TRIM( TRAILING '.' FROM domain )
WHERE domain != '.' AND domain NOT LIKE '%..'
```